plugin/src/main/java/net/mcjukebox/plugin/bukkit/commands/RegionCommand.java

```diff
@@ -25,7 +26,17 @@ public RegionCommand(RegionManager regionManager) {
     public boolean execute(CommandSender dispatcher, String[] args) {
         // region add <id> <url>
         if (args.length == 3 && args[0].equalsIgnoreCase("add")){
-            MCJukebox.getInstance().getRegionManager().addRegion(args[1], args[2]);
+            String url = args[2];
+            
+            if(!UrlUtils.isValidURI(url)) {
+                MessageUtils.sendMessage(dispatcher, "command.invalidUrl");
+                return true;
+            }
+            if(!UrlUtils.isDirectMediaFile(url)) {
```

Comment:
Nit: Can we have a newline between the two if blocks and a space after the `if` keyword?